docs/docs/api/hooks/useFrameCallback.md

```diff
@@ -36,9 +37,18 @@ Properties:
                     or not (`false`)
 * `callbackId: number`: a unique identifier of the callback function
 
+#### `FrameTimings: [object]`
+
+Properties:
+* `timestamp: number`: the current system time
+* `frameTime: number`: time since last frame - this value may be zero on the first frame update
+* `elapsedTime: number`: time since the callback started running
```

Comment:
1. Add info about unit (in seconds or in milliseconds?) in each line for better readability.
2. What do you think about renaming `frameTime` to `timeSinceLastFrame`?
3. I'm wondering if instead of `frameTime = 0` we can have `isFirstFrame = true`.